app/src/main/res/layout-land/onboarding_slide.xml

```diff
@@ -18,20 +17,21 @@
     <ImageView
       android:id="@+id/slide_image_view"
       android:layout_width="0dp"
-      android:adjustViewBounds="true"
-      android:scaleType="fitXY"
       android:layout_height="0dp"
+      android:adjustViewBounds="true"
       android:contentDescription="@{viewModel.contentDescription}"
+      android:scaleType="fitXY"
       android:src="@{viewModel.slideImage}"
       app:layout_constraintBottom_toBottomOf="parent"
-      app:layout_constraintDimensionRatio="H,4:4.5"
+      app:layout_constraintDimensionRatio="4:4"
```

Comment:
The ratio for the image is 10:9
`app:layout_constraintDimensionRatio="H, 10:9"` once this change is done, checkout the circle of smiley face, they look more rounded and works well with mocks.